firestore.rules

```diff
@@ -10,8 +10,8 @@ service cloud.firestore {
         match /items/{itemId} {
 
             allow create: if isSignedInAndSelf(request.resource.data.userId);
-            allow update: if isSignedInAndSelf(resource.data.userId) || (isSignedIn() && request.resource.data.diff(resource.data).affectedKeys()
-                .hasOnly(['sold' , 'rated']));
+            allow update:if isSignedInAndSelf(resource.data.userID) || (isSignedIn() && request.resource.data.diff(resource.data).affectedKeys()
+                    .hasOnly(['quantity','sold','reviews']))
```

Comment:
Why do you have userID instead of userId? Also why two tabs before the .hasOnly?

Reply:
Same questions, how does it still work with `userID` instead of `userId`?

Reply:
Yeah that I have no clue why it works cause I didn't mean to change that part, typo when I rewrote it. I fixed it in the new commit.